controllers/install/binoculars_controller_e2e_test.go

```diff
@@ -36,8 +36,12 @@ var _ = Describe("Armada Operator", func() {
 		It("Kubernetes should create Executor Kubernetes resources", func() {
 			By("Calling the Executor Controller Reconcile function", func() {
 				f, err := utils.CreateTempFile([]byte(binocularsYaml))
-				defer f.Close()
-				defer os.Remove(f.Name())
+				defer func() {
```

Comment:
Could this not be simplified to:

```go
Expect(err).ToNot(HaveOccurred())
defer f.Close()
defer os.Remove(f.Name())
```

Or is that bad practise?

Reply:
I interpreted the warning slightly differently. I thought that its complaining because we are not checking if variable `f` is created without error beforing defering, because this would cause a crash, rather than the defers failing, which wouldn't cause an imediate crash.

Reply:
Well the linter check is passing now.  It was saying that we need to check the err in the defer in case that fails.